app/lib/common/widgets/html_editor/html_editor.dart

```diff
@@ -404,11 +446,28 @@ class HtmlEditorState extends State<HtmlEditor> {
     if (attributes == null) {
       return before;
     }
+
     final roomId = widget.roomId;
     // Inline Mentions
-    final mention = attributes.entries
-        .firstWhere((e) => e.value is MentionAttributes)
-        .value as MentionAttributes?;
+    MentionAttributes? mention;
+    try {
+      final mentionEntry = attributes.entries
+          .where((e) => e.value is MentionAttributes)
+          .firstOrNull;
+      mention = mentionEntry?.value as MentionAttributes?;
```

Comment:
if you import `collection` on the top, you can also just do:
```suggestion
      mention = attributes.entries
          .firstOrNull((e) => e.value is MentionAttributes)
          ?.value as MentionAttributes?;
```

Reply:
this'd announce to use `!` so my way prevents it ;)